src/main/java/seedu/weme/logic/parser/ArgumentTokenizer.java

```diff
@@ -28,6 +30,25 @@ public static ArgumentMultimap tokenize(String argsString, Prefix... prefixes) {
         return extractArguments(argsString, positions);
     }
 
+    /**
+     * Parse the given arguments to obtain the last prefix with its arguments.
+     * @param argsString Arguments string of the form: {@code preamble <prefix>value <prefix>value ...}
+     * @param prefixes   Prefixes to tokenize the arguments string with
+     * @return           LastArgument in the argument string
+     */
+    public static LastArgument getLastArgument(String argsString, Prefix... prefixes) {
+        List<PrefixPosition> prefixPositions = findAllPrefixPositions(argsString, prefixes);
+        if (prefixPositions.isEmpty()) {
+            return null;
+        }
+
+        prefixPositions.sort((prefix1, prefix2) -> prefix2.getStartPosition() - prefix1.getStartPosition());
+        PrefixPosition lastPrefixPosition = prefixPositions.get(0);
+        Prefix lastPrefix = lastPrefixPosition.prefix;
+        PrefixPosition endPositionMarker = new PrefixPosition(new Prefix(""), argsString.length());
```

Comment:
Empty String should also be considered as a magic literal.

Reply:
It is how AB# does it tho. In this case the empty string is just a dummy placeholder.